[Partial of #5483] Zelda Mobile Fixes

dbalders opened the pull request:
Partial of #5483

Fixes the following in the list:
* no mobile menu on about page
    * Was missing the code entirely so added it to the `about.hbs`
* sidebar nav items too small on mobile
    * Changed font from 1.3em to 1.6em for < 500px
* main header too big
    * Changed from 65px to 50px

@JohnONolan: I expect you to want to make some changes, so just let me know if you want anything different or changed in some way.

Screenshots after my changes:
![img_0686](https://cloud.githubusercontent.com/assets/4715098/8345277/83d7165a-1aa1-11e5-91be-ddffceb25680.PNG)
![img_0687](https://cloud.githubusercontent.com/assets/4715098/8345281/89dc766c-1aa1-11e5-964c-5fde45f253bb.PNG)

JohnONolan commented:
Yo, appreciate the help! I would definitely recommend attempting one bullet at a time, however - as this has quite a few problems which makes it harder to sort though.

- Overflows and text-overflows all exist for a reason, they can't just be taken off. They're all needed to allow long strings and content to be handled gracefully
- HTML files should never be modified, they're prototypes with all sorts of hacks in them which are not supposed to stay up to date
- Hit area on mobile button needs expanding both horizontally and vertically and to be optimised specifically for touch interactions
- Dropdown mobile positioning is a whole other thing
- Font size in nav menu is only meant to increase on mobile view, not by default

dbalders commented:
@JohnONolan: cool, thanks! I updated based on what you said, but pulled out a few things from the pr (dropdown mobile positioning, overflows, mobile button hit area) and will take another shot in their own pr's. I have updated the original description of the pr to reflect this.

dbalders commented:
@JohnONolan just thought i would ping you again. Happy to continue down the list of other things in parent issue as well if you would like.
